tests/normal.t.pl

```diff
@@ -28,10 +28,10 @@ sub runTests
     my $hardlinks = 1;
     if($mode eq 'standard')
     {
-        &mount("--standard");
+        &mount("--standard -o direct_io");
     } elsif($mode eq 'paranoia')
     {
-        &mount("--paranoia");
+        &mount("--paranoia -o direct_io");
```

Comment:
So I was OK, I patched FreeBSD fuse adding the 3 previous lines of code, and it works.
I'll then correct this PR accordingly and open a FreeBSD issue.